common/imageloading/build.gradle.kts

```diff
@@ -24,7 +23,8 @@ kotlin {
 
         implementation(libs.kotlininject.runtime)
 
-        api(libs.imageloader)
+        api(libs.coil.core)
+        api(libs.coil.network)
```

Comment:
You'll need to import a Ktor engine for each platform as well. Here's what I use: https://github.com/coil-kt/coil/blob/main/samples/shared/build.gradle.kts#L44

Reply:
There's no huge need here, as the Ktor engines are already on the app classpath.